codes/go/chapter_tree/array_binary_tree_test.go

```diff
@@ -0,0 +1,48 @@
+// File: array_binary_tree_test.go
+// Created Time: 2023-07-24
+// Author: Reanon (793584285@qq.com)
+
+package chapter_tree
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/krahets/hello-algo/pkg"
+)
+
+func TestArrayBinaryTree(t *testing.T) {
+	// 初始化二叉树
+	// 这里借助了一个从数组直接生成二叉树的函数
+	arr := []any{1, 2, 3, 4, 0, 6, 7, 8, 9, 0, 0, 12, 0, 0, 15}
```

Comment:
Can we replace 0 with nil?

Reply:
If not, I think INT_MAX is a better choice.

Reply:
done